app/vmauth/README.md

```diff
@@ -280,6 +289,8 @@ See the docs at https://docs.victoriametrics.com/vmauth.html .
      Per-second limit on the number of WARN messages. If more than the given number of warns are emitted per second, then the remaining warns are suppressed. Zero values disable the rate limit
   -maxIdleConnsPerBackend int
      The maximum number of idle connections vmauth can open per each backend host (default 100)
+  -maxProxiedConnections int
```

Comment:
It is better naming this flag `-maxConcurrentRequests`, since it limits the number of **concurrent** requests

Reply:
removed global flag, used only per user